fleet-management-spring-mvc/src/main/webapp/WEB-INF/jsp/technician/vehicleDetailView.jsp

```diff
@@ -61,13 +61,12 @@
         </tr>
         </thead>
         <tbody>
-        <c:forEach items="${inspections}" var="inspection">
-        <tr>
-            <td>${inpection.name}</td>
-            <td>${inpection.days}</td>
-
-        </tr>
-    </c:forEach>
+            <c:forEach items="${inspections}" var="inspection">
+            <tr>
+                <td><c:out value="${inspection.name}"/></td>
```

Comment:
What is actually the difference between using the `<c:out value="${inspection.name}"/>` and just `${inspection.name}`?